src/ui/reducers/timeline.ts

```diff
@@ -71,6 +71,8 @@ const timelineSlice = createSlice({
   },
 });
 
+const EPSILON = 0.0001;
```

Comment:
What is this?

I don't think this word has a standalone meaning that provides enough context into what it is.

Reply:
Could maybe rename this to be a bit clearer, but "Epsilon" is a standard term for "amount of tolerance when comparing floats":

- https://en.wikipedia.org/wiki/Machine_epsilon

Reply:
Yeah, that's the meaning I was using, but you're right, it's not obvious to people who have not done floating point stuff much.